libweston/backend-rdp/rdprail.c

```diff
@@ -404,12 +412,21 @@ rail_client_WindowMove_callback(bool freeOnly, void *arg)
 	if (!freeOnly)
 		surface = (struct weston_surface *)rdp_id_manager_lookup(&peerCtx->windowId, windowMove->windowId);
 	if (surface) {
+		rail_state = (struct weston_surface_rail_state *)surface->backend_state;

```

Comment:
for upstream we don't typecast void pointers

Reply:
Changed.